### PR TITLE
Plane: log EAS2TAS in CTUN

### PR DESCRIPTION
**Pull request by IamPete1**
Adds logging of airspeed ratio to TECS, useful to understand the difference between readings from sensor that used by TECS.

### CODE REVIEW

**Review by magicrub — APPROVED** [review]

File: libraries/AP_TECS/AP_TECS.cpp [hunk 1]
```diff
@@ -1220,15 +1220,17 @@ void AP_TECS::update_pitch_throttle(int32_t hgt_dem_cm,
     // @Field: PErr: difference between estimated potential energy and desired potential energy
     // @Field: EDelta: current error in speed/balance weighting
     // @Field: LF: aerodynamic load factor
-    AP::logger().Write("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF",
-                       "s------",
-                       "F------",
-                       "Qffffff",
+    // @Field: ASraito: equivelent to true airspeed ratio
+    AP::logger().Write("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF,ASraito",
```
Comment (peterbarker) [inline libraries/AP_TECS/AP_TECS.cpp:1224]: Why not EAS2TAS?  same length, and it's what people kind of know this as.  It also conveys more information as "airspeed raito" (sic.) isn't really that descriptive.

I mean, the label is mispelled here, so it does need fixing anyway :-)

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: libraries/AP_TECS/AP_TECS.cpp (hunk 1 above)
Comment (IamPete1) [reply libraries/AP_TECS/AP_TECS.cpp]: Opps, I had already fixed the same typo in the commit message too.

Have have changed to `EAS2TAS` for some reason i thought it was longer. Your right that is better.

Thanks

File: libraries/AP_TECS/AP_TECS.cpp [hunk 2]
```diff
@@ -1220,15 +1220,17 @@ void AP_TECS::update_pitch_throttle(int32_t hgt_dem_cm,
     // @Field: PErr: difference between estimated potential energy and desired potential energy
     // @Field: EDelta: current error in speed/balance weighting
     // @Field: LF: aerodynamic load factor
-    AP::logger().Write("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF",
-                       "s------",
-                       "F------",
-                       "Qffffff",
+    // @Field: EAS2TAS: equivelent to true airspeed ratio
+    AP::logger().Write("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF,EAS2TAS",
```
Comment (tridge) [inline libraries/AP_TECS/AP_TECS.cpp:1224]: this is useful in non-TECS modes, such as FBWA, so CTUN seems like a good place

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/AP_TECS/AP_TECS.cpp (hunk 2 above)
Comment (IamPete1) [reply libraries/AP_TECS/AP_TECS.cpp]: moved to CTUN

**Comment by IamPete1** [comment]
Moved to CTUN, required some fairly ruthless shortening of the labels to get them to all fit

**Comment by peterbarker** [comment]
Changing labels isn't free - any pre-canned graphs get broken.

If we're going to break things, we should go to `ThO` and `ThD` - we already use the former somewhere else and the latter would seem to follow.

Plenty of room in `BARO` :-P

**Comment by IamPete1** [comment]
Seems a shame to put in baro, we only need it on plane and only for the primary. But open to ideas. 

I did ponder about removing roll and pitch (both nav and not) from CTUN. There in all in ATT anyway. Throttle and rudder out are also in both in ATER. Then we would have loads of room.

**Review by tridge — APPROVED** [review]

**Review by tridge — APPROVED** [review]

File: ArduPlane/Log.cpp [hunk 3]
```diff
@@ -300,14 +302,15 @@ const struct LogStructure Plane::log_structure[] = {
...
 // @Field: Pitch: achieved pitch
-// @Field: ThrOut: scaled output throttle
+// @Field: ThO: scaled output throttle
 // @Field: RdrOut: scaled output rudder
-// @Field: ThrDem: demanded speed-height-controller throttle
-// @Field: Aspd: airspeed estimate (or measurement if airspeed sensor healthy and ARSPD_USE>0)
+// @Field: ThD: demanded speed-height-controller throttle
+// @Field: As: airspeed estimate (or measurement if airspeed sensor healthy and ARSPD_USE>0)
 // @Field: SAs: synthetic airspeed measurement derived from non-airspeed sensors, NaN if not available
+// @Field: E2T: equivelent to true airspeed ratio
```
Comment (peterbarker) [inline ArduPlane/Log.cpp:310]: Spelting error here.
```suggestion
// @Field: E2T: equivalent to true airspeed ratio
```

**Review by peterbarker — APPROVED** [review]

**Comment by peterbarker** [comment]
PR for spelting error would be nice but not essential.

**Pull request merged**